cli/tests/activate/attach.exp

```diff
@@ -20,6 +20,7 @@ expect_after {
 }
 
 expect "Attached to existing activation"
+expect "To stop using this environment, type 'exit'"
```

Comment:
Would expecting on the prompt, now that we set it in 0a9a978805dcf2c53bbbb0fb312d432ee8486743, also work and be more reliable since we know that's when it's actually ready to receive a command?

Reply:
Okay I expected on the prompt...and made the same change everywhere else. And dropped a few unnecessary .exp files